test/expression_callbacks_test.exs

```diff
@@ -1,5 +1,48 @@
 defmodule ExpressionCallbacksTest do
+  @moduledoc false
   use ExUnit.Case, async: true
   doctest Expression.Callbacks, import: true
   doctest Expression.Callbacks.Standard, import: true
+
+  @context %{
+    "base_date" => ~U[2023-02-03 20:18:03Z],
+    "number" => %{"from_addr" => "+552197295926"}
+  }
+
+  describe "next/4" do
+    test "Shifts a datetime to the next occurrence of a given day of the week and a set time." do
+      assert Expression.evaluate_block!(~s(next\(monday, 10:00, base_date\)), @context) ==
```

Comment:
How is this working when `monday` here is not a quoted string? 
My guess is that it should be something like
```elixir
Expression.evaluate_block(~s[next("monday", "10:00", base_date)], ctx) 
```